package.json

```diff
@@ -10,6 +10,7 @@
     "prettier-plugin-solidity": "^1.0.0-alpha.12"
   },
   "scripts": {
+    "publish": "npx lerna run clean:3rd-parties --scope tasit-contracts && npx lerna publish",
```

Comment:
`npx lerna publish` worked for you directly? I know before we had been using `npx lerna public from-package`

What prompts did you get when going this route?

Reply:
That works. After the run, shows possible versions, I've selected patch 0.0.7 and then as final prompt a list of packages to upgrade.